src/frontend/src/pages/AdminToolsPage/RecruitmentConfig/FAQTable.tsx

```diff
@@ -1,25 +1,42 @@
+import React, { useState } from 'react';
 import { TableRow, TableCell, Box, Table as MuiTable, TableHead, TableBody, Typography, Button } from '@mui/material';
 import EditIcon from '@mui/icons-material/Edit';
 import DeleteIcon from '@mui/icons-material/Delete';
 import { FrequentlyAskedQuestion } from 'shared/src/types/frequently-asked-questions-types';
 import { NERButton } from '../../../components/NERButton';
-import { useAllFaqs } from '../../../hooks/recruitment.hooks';
+import { useAllFaqs, useDeleteFAQ } from '../../../hooks/recruitment.hooks';
 import LoadingIndicator from '../../../components/LoadingIndicator';
 import { useHistoryState } from '../../../hooks/misc.hooks';
 import ErrorPage from '../../ErrorPage';
 import CreateFaqFormModal from './CreateFaqFormModal';
 import EditFaqFormModal from './EditFaqFormModal';
+import NERDeleteModal from '../../../components/NERDeleteModal';
+import { useToast } from '../../../hooks/toasts.hooks';
 
 const FAQsTable = () => {
   const [createModalShow, setCreateModalShow] = useHistoryState<boolean>('', false);
   const [faqEditing, setFaqEditing] = useHistoryState<FrequentlyAskedQuestion | undefined>('', undefined);
+  const [faqToDelete, setFaqToDelete] = useState<FrequentlyAskedQuestion | undefined>(undefined);
+  const { mutateAsync: deleteFaq } = useDeleteFAQ();
+  const toast = useToast();
+
   const { isLoading: faqsIsLoading, isError: faqsIsError, error: faqsError, data: faqs } = useAllFaqs();
+  const handleDelete = (id: string) => {
+    try {
+      deleteFaq(id);
+      setFaqToDelete(undefined);
```

Comment:
Move this outside of the `try-catch` so the modal closes even if there is a failure